openassessment/fileupload/backends/django_storage.py

```diff
@@ -28,7 +30,11 @@ def get_download_url(self, key):
         """
         path = self._get_file_path(key)
         if default_storage.exists(path):
-            return default_storage.url(path)
+            storage_path = default_storage.url(path)
+
+            # Return a fully-qualified URL
+            lms_url = settings.LMS_ROOT_URL
```

Comment:
would it complain if you had `getattr(settings, 'LMS_ROOT_URL', '')`? (the added default) That feels like a better way and I'd argue we should just add a pylint ignore if that makes it complain.

Reply:
We use all 3 paradigms (`settings.LMS_ROOT_URL`, `getattr` with and w/out default) and it complains about the `getattr` implementations. Have pylint ignored.